fcs.tex

```diff
@@ -1,248 +0,0 @@
-\glspl{NFCS} are system-level analysis tools
```

Comment:
why has this file been deleted?

Reply:
the contents in this file have been moved to the introduction, for better flow. 
https://github.com/jbae11/master_thesis/blob/edits/introduction.tex#L263